### ISSUE TITLE
AWESOME project.!

### ISSUE THREAD

**Issue by inglesuniversal** [kevinchappell/formBuilder#871]
How can I contribute?

Personal Strengths: 

* Ideas / Creativity
* English / Spanish
* Educational Field
* English Learning Centers
* Online Training
* Website / App Design
* FullStack Web & App Developer
* Over 35 years of experience 
* Cloud Storage Service

**Comment by kevinchappell** [comment]
@inglesuniversal Thanks for the kind words, formBuilder can use all the help it can get. If you use the project and find any area that could use improvement a PR is more than appreciated.

Areas that could use improvement:
- tests. formBuilder desperately needs tests implemented for peace of mind and to prevent regression issues when new features are added or existing features enhanced.
- documentation. I spent a lot of time updating the docs for 2.10.0 but there is always work to be done here. 
  - One of the most common things that is asked is how to save data to a database. This is a subject I have intentionally avoided for years because the correct answer is that it depends on tech stack and framework and any doc recommending a method would likely be wrong for the majority of people reading it. For example if a doc recommends a PHP way of receiving the form submission and saving to MySQL- it would likely not apply to a WordPress based app where the endpoint is an admin ajax endpoint or the app is Ruby using MongoDB. That said, I would be happy to include docs for the most popular use cases on the site as long as it provides value but there should maybe be a disclaimer that backend recommendations should not be copy pasted and serve only as a high level guide.
- bugfixes. always help needed here.
- website enhancements. The website would be a lot more useful if users could save and retrieve form templates using blockchain, generate configurations and make translating easier. I have been thinking to open source the website to help build out some of these features. WDYT?

A good start would be to browse the issues and jump into threads that look like they need help. In the last 3 weeks about 150 issues were closed or resolved but there are some older issues I haven't had time to address yet.
